package.json

```diff
@@ -50,6 +50,7 @@
     "generate-api-doc": "scripty",
     "parse-log": "node ./dist/scripts/parse-log.js",
     "prune-storage": "node ./dist/scripts/prune-storage.js",
+    "optimize-old-videos": "node ./dist/scripts/optimize-old-videos.js",
```

Comment:
You forgot to add the script source.

Reply:
No idea where that went. Luckily I still had it open in the editor.